### DISCUSSION TITLE
Logical Nodes in the Gating Strategy

### DISCUSSION THREAD

**Discussion by brianbourke75** [whitews/FlowKit#222] [General]
I was wondering if there are any plans to add support for logical nodes (OrNode, AndNode, etc) in the gating strategy?   I am attempting to mine data and while the library works like a charm, I ran into a data set which has these, and I may need these supported.

EDIT: Never mind... just found issue issue #179 :)

**Comment by whitews** [comment]
Hi Brian,

Yes, seems like you found it but the Boolean gates in FlowJo workspaces will be in the upcoming release. I'm in the process of updating the documentation now. Version 1.2.0 should be out in the next few days and available to pip install.

Thanks for using FlowKit!
-Scott
